addons/binding/org.openhab.binding.freebox/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,233 @@
+	</channel-type>
+	
+	<channel-type id="tempcpum">
+		<item-type>Number</item-type>
+		<label>CPUm Temperature</label>
+		<description>Actual measured CPU Marvell temperature of the Freebox Server</description>
+	</channel-type>
+	
+	<channel-type id="tempcpub">
+		<item-type>Number</item-type>
+		<label>CPUb Temperature</label>
+		<description>Actual measured CPU Broadcom (xDSL) temperature of the Freebox Server</description>
+	</channel-type>
+	
+	<channel-type id="tempswitch">
+		<item-type>Number</item-type>
+		<label>Switch Temperature</label>
+		<description>Actual measured switch temperature of the Freebox Server</description>
+	</channel-type>
+	
+	<channel-type id="fanspeed">
+		<item-type>Number</item-type>
+		<label>Fan Speed</label>
+		<description>Actual measured fan speed (rpm) of the Freebox Server</description>
+	</channel-type>
+	
+	<channel-type id="lcd_brightness">
+		<item-type>Number</item-type>
+		<label>Screen Brightness</label>
+		<description>Brightness level of the screen in percent</description>
+	</channel-type>
+	
+	<channel-type id="lcd_orientation">
+		<item-type>Number</item-type>
+		<label>Screen Orientation</label>
+		<description>Orientation de l'écran</description>
+	</channel-type>
+	
+	<channel-type id="lcd_forced">
+		<item-type>Switch</item-type>
+		<label>Forced Orientation</label>
+		<description>Orientation de l'écran forcée</description>
+	</channel-type>
+	
+	<channel-type id="wifi_status">
+		<item-type>Switch</item-type>
+		<label>Wifi Enabled</label>
+		<description>Connexion Wifi activée</description>
+	</channel-type>
+	
+	<channel-type id="xdsl_status">
+		<item-type>String</item-type>
+		<label>xDSL Status</label>
+		<description>Status of the xDSL line</description>
+	</channel-type>
+	
+	<channel-type id="line_status">
+		<item-type>String</item-type>
+		<label>Line Status</label>
+		<description>Status of network line connexion</description>
+	</channel-type>
+	
+	<channel-type id="ipv4">
+		<item-type>String</item-type>
+		<label>IP Address</label>
+		<description>Public IP Address of the Freebox Server</description>
+	</channel-type>
+	
+	<channel-type id="rate_up">
+		<item-type>Number</item-type>
+		<label>Upload Rate</label>
+		<description>Current upload rate in byte/s</description>
+	</channel-type>
+	
+	<channel-type id="rate_down">
+		<item-type>Number</item-type>
+		<label>Download Rate</label>
+		<description>Current download rate in byte/s</description>
+	</channel-type>
+	
+	<channel-type id="bytes_up">
+		<item-type>Number</item-type>
+		<label>Uploaded</label>
+		<description>Total uploaded bytes since last connection</description>
+	</channel-type>
+	
+	<channel-type id="bytes_down">
+		<item-type>Number</item-type>
+		<label>Downloaded</label>
+		<description>Total downloaded bytes since last connection</description>
+	</channel-type>
+	
+	<channel-type id="onhook">
+		<item-type>Switch</item-type>
+		<label>Raccroché</label>
```

Comment:
Could you please translate the text to english? Note that you can provide french translations in the i18n folder.